### ISSUE TITLE
Dependencies in Fedora

### ISSUE THREAD

**Issue by storrgie** [yktoo/ymuse#36]
The two dependencies of the project:
 - `libc6`
 - `libgtk-3-0`

Don't appear to exist in Fedora (dnf doesn't automatically resolve them when installing the `.rpm`).

Is ymuse intended to be 32-bit? Do I need to install the `.i686` equivalent packages?

For libc6:
```
glibc.x86_64 : The GNU libc libraries
glibc.i686 : The GNU libc libraries
```

For `libgtk-3-0`:
```
gtk3-devel.i686 : Development files for GTK+
gtk3-devel.x86_64 : Development files for GTK+
```

**Comment by storrgie** [comment]
It appears that if I install `glibc.x86_64` and `gtk3-devel.x86_64` it works properly by running the built binary (e.g. `./ymuse`).

**Comment by yktoo** [comment]
The dependency requirements for `.rpm` are indeed a bit off and need to be corrected.

In the meantime, you can install Ymuse in Fedora from `.rpm` with:

```bash
sudo rpm -i --nodeps ymuse_*.rpm
```

`glibc` and `gtk3` are normally already included in Fedora Workstation so there's no additional action required.

**Comment by pietryszak** [comment]
I create little one liner to download and install the newest version of rpm.
```bash
 URL=$(curl -L -s https://api.github.com/repos/yktoo/ymuse/releases/latest | grep -o -E "https://(.*)ymuse_(.*)_linux_amd64.rpm") | wget $URL && mv ymuse*.rpm ymuse.rpm && sudo rpm -i --nodeps  ymuse.rpm && rm ymuse.rpm
```

Thanks for idea [One Liner by knadh](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8#gistcomment-3868412)

**Comment by yktoo** [comment]
If you have `jq` installed, you can use this (and avoid using `wget`):

```bash
curl -fsSL https://api.github.com/repos/yktoo/ymuse/releases/latest | jq -r '.assets[] | select(.name | test("ymuse.*_linux_amd64\\.rpm")) | .browser_download_url' | xargs curl -fsSL -o /tmp/ymuse.rpm && sudo rpm -i --nodeps /tmp/ymuse.rpm && rm /tmp/ymuse.rpm
```

This doesn't, however, address the original dependency-related issue.

**Comment by yktoo** [comment]
Closed via #46.
